bodyfetcher.py

```diff
@@ -233,12 +291,39 @@ def make_api_call_for_site(self, site):
                         owner_link = ""
                         owner_rep = 0
 
-                    is_spam, reason, why = check_if_spam(answer_title, body, owner_name, owner_link, site, a_id, True, False, owner_rep, post_score)
+                    is_spam, reason, why = check_if_spam(
+                        answer_title,
+                        body,
+                        owner_name,
+                        owner_link,
+                        site,
+                        a_id,
+                        True,
+                        False,
+                        owner_rep,
+                        post_score
+                    )
                     if is_spam:
                         try:
-                            handle_spam(title, body, owner_name, site, link, owner_link, a_id, reason, True, why, owner_rep, post_score, up_vote_count, down_vote_count, q_id)
+                            handle_spam(
+                            title,
```

Comment:
You've got weird indentation here - arguments should be indented from the method name if you're using arguments on new lines; otherwise (see hichris' comment) they should just be on the same line as the method call.